sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/events.scala

```diff
@@ -62,6 +62,16 @@ case class DropDatabasePreEvent(database: String) extends DatabaseEvent
 case class DropDatabaseEvent(database: String) extends DatabaseEvent
 
 /**
+ * Event fired before a database is altered.
+ */
+case class AlterDatabasePreEvent(database: String) extends DatabaseEvent
```

Comment:
Hi, @jerryshao .
We are adding `AlterTableEvent` and `AlterDatabaseEvent`. Can we have a more specific PR title instead of `Add more ExternalCatalogEvent`?

Reply:
Sure, I will update the title.